Neo4j.Driver/common.props

```diff
@@ -1,10 +1,10 @@
 <Project>
     <PropertyGroup>
         <Copyright>Copyright 2002-2023</Copyright>
```

Comment:
Unrelated: should this also be bumped at some point?

Reply:
Yeah probably.